### PR TITLE
ADCM-6134: Implement node for getting actions

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: adcm_aio_client/core/objects/cm.py [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
 
 type Filter = object  # TODO: implement
 
 
+# TODO: Avoiding circular imports. Think about right place for such mixins.
+class WithActions(WithRequester, AwareOfOwnPath):
+    @cached_property
+    def actions(self: Self) -> "ActionsAccessor":
+        return ActionsAccessor(
+            parent=self,  # pyright: ignore [reportArgumentType]
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:34]: why's ignore? better rule that out now

looks like too much restrictions on parent or too little inheritance on WithActions

File: adcm_aio_client/core/objects/cm.py [hunk 2]
```diff
@@ -297,3 +293,54 @@ async def _get_hosts_from_arg_or_filter(
...
+    @cached_property
+    def display_name(self: Self) -> str:
+        return self._data["displayName"]
+
+    async def run(self: Self) -> dict:  # TODO: implement Task, return Task
+        return (await self._requester.post(*self.get_own_path(), "run", data={"isVerbose": self._verbose})).as_dict()
+
+    @async_cached_property
+    async def _mapping_rule(self: Self) -> list[dict]:
+        return (await self._rich_data)["hostComponentMapRules"]
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:318]: is it used somewhere?

File: adcm_aio_client/core/objects/cm.py [hunk 3]
```diff
@@ -297,3 +293,54 @@ async def _get_hosts_from_arg_or_filter(
...
+
+    @cached_property
+    def mapping(self: Self) -> "ActionMapping":
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:321]: make it async from the start, because it requires full object

Comment (DanBalalan) [reply adcm_aio_client/core/objects/cm.py]: done

File: adcm_aio_client/core/objects/cm.py (hunk 2 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/cm.py:318]: No, but this field is [described](https://wiki.adsw.io/books/adcm/page/adcm-aio-client#bkmrk-action-attributes%3A-i) in Ation's attributes

File: adcm_aio_client/core/objects/cm.py (hunk 1 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/cm.py]: refactored

**Review by a-alferov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Pull request merged**